Source/WebDriver/WebDriverService.cpp

```diff
@@ -52,12 +69,18 @@ static void printUsageStatement(const char* programName)
     printf("  -p <port>,   --port=<port>      Port number the driver will use\n");
     printf("               --host=<host>      Host IP the driver will use, or either 'local' or 'all' (default: 'local')\n");
     printf("  -t <ip:port> --target=<ip:port> Target IP and port\n");
+#if ENABLE(WEBDRIVER_BIDI)
+    printf("               --bidi-port=<port>        Port number to use for BiDi's WebSocket connections\n");
+#endif
 }
 
 int WebDriverService::run(int argc, char** argv)
 {
     String portString;
     std::optional<String> host;
+#if ENABLE(WEBDRIVER_BIDI)
+    String bidiPortString;
```

Comment:
Nit: use member initializers

Reply:
Given those are local variables to the `run` method, do you mean default initializing them?